src/testdrive/parser.rs

```diff
@@ -120,16 +136,37 @@ fn parse_sql(line_reader: &mut LineReader) -> Result<SqlCommand, InputError> {
                 expected_rows.push(split_line(pos3, &line3)?);
             }
         }
-        (Some((pos2, line2)), None) => expected_rows.push(split_line(pos2, &line2)?),
+        (Some((pos2, line2)), None) => match HASH_REGEX.captures(&line2) {
+            Some(captures) => match captures.get(1).unwrap().as_str().parse::<usize>() {
+                Ok(num_values) => {
+                    return Ok(SqlCommand {
+                        query,
+                        expected_result: SqlExpectedResult::Hashed {
+                            num_values,
+                            md5: captures.get(2).unwrap().as_str().to_owned(),
+                        },
+                    })
+                }
+                Err(_) => {
+                    return Err(InputError {
+                        pos: pos2,
+                        msg: "Invalid value for number of expected rows".into(),
```

Comment:
Would be nice to keep the underlying error here: `msg: format!("invalid value for number of expected rows: {}", e)`